lib/helpers/project-init.js

```diff
@@ -33,7 +33,7 @@ module.exports = function(options) {
   var presets = initPresets[target]();
   var projectName = (target === 'addon') ? 'my-addon' : 'my-app';
   var cliOptions = {
-    package: options.package || path.resolve(process.cwd(), '..', '..', 'tests', 'fixtures', type, 'package'),
+    package: options.package || path.resolve(process.cwd(), '..', '..', 'node-tests', 'fixtures', type, 'package'),
```

Comment:
is this actually process.cwd() ? Often CLI libs can be invoked from sub dirs, at which this wont be correct.